galley/pkg/server/components/processing2.go

```diff
@@ -0,0 +1,346 @@
+// Copyright 2018 Istio Authors
```

Comment:
this file is named `processing2.go`?  I assume there is already a file called `processing.go`?  Would it be better to rename the old one to `processing_old.go`?

Reply:
Sure. I can do that in a separate check-in. We have used the "*2" paradigm in a couple of instances when doing similar things (e.g. in Mixer), so it should be well understood.

Reply:
Actually, I take that back. I'll keep it as is until this merges to master, to avoid gnarly merge issues.